src/test/java/io/confluent/connect/jdbc/sink/integration/JdbcSinkConnectorIT.java

```diff
@@ -0,0 +1,269 @@
+  private Map<String, String> props;
+  private static Connection connection;
+
+  @ClassRule
+  public static DockerComposeContainer mySqlContainer =
+      new DockerComposeContainer(new File("src/test/docker/configA/mysql-docker-compose.yml"));
+
+  public static DockerComposeContainer pumbaContainer;
+  private void startPumbaContainer() {
+    pumbaContainer =
+        new DockerComposeContainer(new File("src/test/docker/configB/pumba-docker-compose.yml"));
+    pumbaContainer.start();
+  }
+
+  @Before
+  public void setup() throws Exception {
+    startConnect();
+    connect.kafka().createTopic(KAFKA_TOPIC, 1);
+    props = getCommonProps();
+    if (connection == null) {
+      TestUtils.waitForCondition(() -> assertConnection().orElse(false),
+          TimeUnit.SECONDS.toMillis(50),
+          "Failed to start the container.");
+    }
+  }
+
+  private Optional<Boolean> assertConnection() {
+    try {
+      connection = getConnection();
+      if (connection != null) {
+        return Optional.of(true);
+      }
+      //Delay to avoid frequent connection requests.
+      TimeUnit.MILLISECONDS.sleep(100);
+      return Optional.empty();
+    } catch (SQLException | InterruptedException e) {
+      return Optional.empty();
+    }
+  }
+
+  @After
+  public void close() {
+    // delete connector
+    connect.deleteConnector(CONNECTOR_NAME);
+    connect.stop();
+  }
+
+  @AfterClass
+  public static void closeConnection() {
+    mySqlContainer.close();
+  }
+
+  @Test
+  public void testWithJDBCMysqlDialectSuccess() throws Exception {
+    dropTableIfExists(KAFKA_TOPIC);
+    sendTestDataToKafka();
+    ConsumerRecords<byte[], byte[]> totalRecords = connect.kafka().consume(
+        NUM_RECORDS,
+        CONSUME_MAX_DURATION_MS,
+        KAFKA_TOPIC);
+    log.info("Number of records added in kafka {}", totalRecords.count());
+
+    // Add mysql dialect related configurations.
+    getConnectorConfigurations();
+
+    // Configure Connector and wait some specific time to start the connector.
+    connect.configureConnector(CONNECTOR_NAME, props);
+    waitForConnectorToStart(CONNECTOR_NAME, Integer.valueOf(MAX_TASKS));
+
+    // Wait Connector to write data into Mysql
+    waitConnectorToWriteDataIntoMysql(
+        connection,
+        CONNECTOR_NAME,
+        Integer.valueOf(MAX_TASKS),
+        KAFKA_TOPIC,
+        NUM_RECORDS);
+
+    int count = loadFromSQL(KAFKA_TOPIC);
+    Assert.assertEquals(NUM_RECORDS, count);
+
+    sendTestDataToKafka();
+    totalRecords = connect.kafka().consume(
+        NUM_RECORDS * 2,
+        CONSUME_MAX_DURATION_MS,
+        KAFKA_TOPIC);
+    log.info("Number of records added in kafka {}", totalRecords.count());
+    // Wait Connector to write data into Mysql
+    waitConnectorToWriteDataIntoMysql(
+        connection,
+        CONNECTOR_NAME,
+        Integer.valueOf(MAX_TASKS),
+        KAFKA_TOPIC,
+        NUM_RECORDS * 2);
+    count = loadFromSQL(KAFKA_TOPIC);
```

Comment:
not required.
To the count assertion and the content assertion in a single method.